data/items.dat

```diff
@@ -88,10 +88,17 @@ build_list = ALL
 [Fusion Reactor]
 cost_list = 50000 | 0 | 9
 type = reactor
-quality_list = discover_modifier | 2500
+quality_list = discover_modifier | 2200
 pre = Fusion Reactor
```

Comment:
why does this have itself as prereq ?

Reply:
No, Fusion Reactor is a tech. And a item.

Reply:
To clarify, Fusion Reactor the item depends of Fusion Reactor the tech.

Reply:
Maybe we then want to make the tech 'Fusion' and 'Advanced Fusion' to make this less confusing?  (Not required for this change, just food for thought.)